server/game/cards/locations/04/riverrun.js

```diff
@@ -0,0 +1,35 @@
+const DrawCard = require('../../../drawcard.js');
+
+class Riverrun extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardPowerChanged: (event, card, power) => {
+                    if(!this.kneeled
+                       && card.getType() === 'character' && card.hasTrait('House Tully')
+                       && power > 0) {
+                        this.tullyCharacter = card;
+
+                        return true;
+                    }
+                    return false;
+                }
+            },
+            handler: () => {
+                if(!this.tullyCharacter) {
+                    return false;
+                }
+                this.controller.kneelCard(this);
```

Comment:
Use the new `cost` property instead (and then you don't need to check !this.kneeled in the event). https://github.com/cryogen/throneteki/blob/master/docs/implementing-cards.md#paying-additional-costs-for-reactions-and-interrupts